packages/button/stories/Button.stories.mdx

```diff
@@ -0,0 +1,154 @@
+import { Add, ExpandMore } from '@manifest-ui/icons';
+import { Box, Flex } from '@manifest-ui/layout';
+import { ExternalLink, Meta, Props, Sources, Story } from '../../../.storybook/components';
+import { Button } from '../src';
+
+<Meta component={Button} title="Components/Button" />
+
+# Button
+
+<ExternalLink href="https://p44design.systems/27d9b4710/p/33fb24-button/b/9368c7">
```

Comment:
I have an opinion to remove this. Or how might we continue to maintain linking as we continue to version? Will this get out of hand or out of date quickly?